lib/Mojolicious/Plugin/AssetPack.pm

```diff
@@ -198,7 +198,9 @@ sub _tag_helpers {
   return $assets->grep(sub { !$_->isa('Mojolicious::Plugin::AssetPack::Asset::Null') })
     ->map(
     sub {
-      my $tag_helper = $_->format eq 'js' ? 'javascript' : 'stylesheet';
+      my $tag_helper = $_->format eq 'js'  ? 'javascript' :
+                       $_->format eq 'png' ? 'image'      :
+                                             'stylesheet';
```

Comment:
Every new supported format needs this code modified to associate format and tag helper...